manifests/09_deployment.yaml

```diff
@@ -51,6 +51,10 @@ spec:
             value: quay.io/openshift/origin-aws-ebs-csi-driver-operator:latest
           - name: AWS_EBS_DRIVER_IMAGE
             value: quay.io/openshift/origin-aws-ebs-csi-driver:latest
+          - name: GCP_PD_DRIVER_OPERATOR_IMAGE
+            value: quay.io/huffmanca/gcp-operator:latest
+          - name: GCP_PD_DRIVER_IMAGE
+            value: quay.io/huffmanca/gcp:latest
```

Comment:
You can use `quay.io/openshift` here. This will be replaced by the cluster's own image.